addons/skin.estouchy/xml/Includes.xml

```diff
@@ -1053,7 +1053,7 @@
 		</item>
 		<item id="5">
 			<label>3</label>
-			<onclick>ActivateWindow(Videos)</onclick>
+			<onclick>ActivateWindow(Video,Root,Return)</onclick>
```

Comment:
this one was needed to work around the issue nate found:
https://www.youtube.com/watch?v=llpq2dgywXk
(see: 1:50)

i don't know why ActivateWindow(Videos) doesn't work here..
ActivateWindow(Videos,Root,Return) didn't work either.
it might due to the fact you can't activate a window that's already active.